### DISCUSSION TITLE
Seeed leds left on

### DISCUSSION THREAD

**Discussion by geetee24** [OpenVoiceOS/ovos-buildroot#257] [Q&A] [answered]
OVOS automatically controls the LEDS on my raspberry pi SEEED hat whenever OVOS is speaking to me the bug is that OVO often leaves. The LED turned on when it’s done speaking. How do I disable OVO from controlling the LED when it is speaking to me so that I can ignore this bug.

**Comment by JarbasAl** [comment, marked as answer]
Uninstall the respeaker plugin

**Comment by geetee24** [comment]
how do i do that?   also what other functionality do i lose if i do that?

**Reply by JarbasAl** [reply]
Pip uninstall package_name. (I'm on the phone and can't check the exact name right now, search our GitHub)

You lose the led integration which is what you asked

**Reply by JarbasAl** [reply]
no clue what SEED hat you are using, i assume one of these plugins is what you are using

https://github.com/OpenVoiceOS/ovos-PHAL-plugin-respeaker-4mic
https://github.com/OpenVoiceOS/ovos-PHAL-plugin-respeaker-2mic

**Comment by geetee24** [comment]
i removed seeed 2 mic package and rebooted. now the problem is worse.   When I say Hey MC, I get no LED lite up when it speaks to me nor when it listens.

when I reinstall it, and reboot.   it has the same bug - the BLUE led is often left ON  for ever until I ask OVIS a question.   So OVOS is unusable  since the LED is on throughout the night too

**Reply by JarbasAl** [reply]
You asked how to remove the led integration, ofc the led won't light up if you uninstall it, that was what you asked...

If you found a bug, open an issue in the corresponding repo explaining in detail what the issue is, so anyone can fix it eventually
